src/views/LoginPage.vue

```diff
@@ -1,47 +1,70 @@
-                            id="email-input"
-                            type="text"
-                            v-model="email"
-                        />
-                        <label for="email-input">Email</label>
-                    </div>
-                    <div class="p-float-label">
-                        <Password
-                            id="password-input"
-                            inputClass="w-full"
-                            class="w-full"
-                            v-model="password"
-                            :feedback="false"
-                        />
-                        <label for="password-input">Password</label>
-                    </div>
-                    <Button
-                        :label="`Not ${userType == 'admin' ? 'an' : 'a'} ${
-                            userType == 'admin' ? 'administrator' : userType
-                        }?`"
-                        class="p-button-link align-self-end mb-6 p-0"
-                        @click="toggleMenu"
-                    />
-                    <Menu ref="userTypeMenu" :model="userTypes" :popup="true" />
-                    <Button
-                        label="Log in"
-                        @click="sendLoginRequest"
-                        :loading="isLoading"
-                    />
-                </div>
-            </template>
-        </Card>
-    </div>
+	<div class="flex flex-row justify-content-center align-items-center" style="padding-top: 150px; padding-bottom: 150px; min-height: 100vh; width: 100vw;">
+		<div class="h-auto flex flex-row">
+			<Card class="btw">
+				<template #content>
+					<div class="flex flex-column justify-content-space-between">
+						<div class="p-float-label mb-5">
+							<InputText class="w-full" id="email-input" type="text" v-model="email" />
+							<label for="email-input">Email</label>
+						</div>
+						<div class="p-float-label">
+							<Password
+								id="password-input"
+								inputClass="w-full"
+								class="w-full"
+								v-model="password"
+								:feedback="false"
+								v-on:keyup.enter="sendLoginRequest" />
+							<label for="password-input">Password</label>
+						</div>
+						<Button
+							:label="`Not ${userType == 'admin' ? 'an' : 'a'} ${
+								userType == 'admin' ? 'administrator' : userType
+							}?`"
+							class="p-button-link align-self-end mb-6 p-0"
+							@click="toggleMenu" />
+						<Menu ref="userTypeMenu" :model="userTypes" :popup="true" />
+					</div>
+					<div v-if="isRegistering" class="flex justify-content-center align-items-center h-20rem">
+						<img
+							alt="Doctor"
+							style="object-fit: cover; width: 100%; max-width: 400px; height: auto"
+							:src="require('@/assets/doctor.png')" />
+					</div>
+					<div class="flex flex-column">
+						<Button label="Log in" @click="sendLoginRequest" :loading="isLoading" />
+						<Button
+							v-if="!isRegistering && userType == 'patient'"
```

Comment:
@dvalim Ogólnie dużo w kodzie macie magic stringów i nigdzie ich do stałych nie wyciągacie. Możecie to w jakimś refactorze poprawić

Reply:
Racja, zajmę się tym na dniach